### ISSUE TITLE
Problem with maxHeight

### ISSUE THREAD

**Issue by Tromens** [Pagawa/PgwSlideshow#14]
Hello!
I've just tested your plugin and it's fantastic!
I just had a problem (I don't know if only occurs to me) when I configure the maxHeight at the starting of the program, and there were different sizes of pictures.
To solve that, I had to modify the code at the line number 94 (from V2), when it checks the maxHeight. I had to add an else with this code:

```
 else {
     height = pgwSlideshow.plugin.find('.ps-current').css('height', height);
 }
```

Thanks for creating this!

**Comment by SgtOddball** [comment]
Good catch, had this problem myself when you use thumbnail images for the carousel. 

From testing it's being caused by a 'bug/issue/we're google and don't give a crap' in chrome/safari where the jQuery load event is triggered before the content is loaded. Tried a few different ways of handling this but this is the only one that works reliably so far.

**Comment by stefdelec** [comment]
Same here. Thumbnails and too many pictures.
